content/acra/configuring-maintaining/general-configuration/acra-authmanager.md

```diff
@@ -71,42 +71,148 @@ read an encrypted file with authentication data.
 
   Default is `.acrakeys`.
 
-### HashiCorp Vault
-
-`acra-authmanager` can read `ACRA_MASTER_KEY` from HashiCorp Vault instead of environment variable.
+### Hashicorp Vault
 
 * `--vault_connection_api_string=<url>`
 
-  Connection string (like `https://example.com:8200`) for connecting to HashiCorp Vault.
-  If not specified, `ACRA_MASTER_KEY` environment variable will be used.
+  Connection string (like `http://x.x.x.x:yyyy`) for loading `ACRA_MASTER_KEY` from HashiCorp Vault.
+  Default is empty (`ACRA_MASTER_KEY` environment variable is expected).
 
-* `--vault_secrets_path=<kv-path>`
+* `--vault_secrets_path=<path>`
 
-  Path to KV Secrets directory in Vault used to store `ACRA_MASTER_KEY`.
+  KV Secret Path for reading `ACRA_MASTER_KEY` from HashiCorp Vault.
   Default is `secret/`.
 
-* `--vault_tls_ca_path=<path>`
+* `--vault_tls_transport_enable=<true|false>`
+
+  Turns on/off TLS for connection with vault to `--vault_connection_api_string` endpoint.
+
+  * `true` — turns on
+  * `false` — (default) turns off.
+
+* `--vault_tls_client_auth=<mode>`
+
+  Set authentication mode that will be used for TLS connection with Vault.
+
+  * `0` — do not request client certificate, ignore it if received;
+  * `1` — request client certificate, but don't require it;
+  * `2` — expect to receive at least one certificate to continue the handshake;
+  * `3` — don't require client certificate, but validate it if client actually sent it;
+  * `4` — (default) request and validate client certificate.
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--vault_tls_ca_path=<filename>`
+
+  Path to CA certificate for HashiCorp Vault certificate validation.
+  Default is empty (deprecated since 0.94.0, use `vault_tls_client_ca` instead).
+
+
+* `--vault_tls_client_ca=<filename>`
+
+  Path to additional CA certificate for Vault certificate validation.
```

Comment:
@Zhaars could you please change all `--vault_tls_client*` fields with more clarifing wording, as shown on example above? As I understand, these all are the same params, they should be described similarly.

```
* `--vault_tls_client_ca=<filename>`

  Path to AcraServer TLS certificate's CA certificate for Vault certificate validation (AcraServer works as "client" when communicating with Vault).
  Empty by default.


* `--vault_tls_client_cert=<filename>`

  Path to AcraServer TLS certificate presented to Vault (AcraServer works as "client" when communicating with Vault).
  Empty by default.


* `--vault_tls_client_key=<filename>`

  Path to AcraServer TLS certificate's private key of the TLS certificate presented to Vault (AcraServer works as "client" when communicating with Vault).
  Empty by default.
```